app/Http/Middleware/LocaleMiddleware.php

```diff
@@ -20,21 +20,27 @@ final class LocaleMiddleware
     public function handle(Request $request, Closure $next): Response
     {
         $user = Auth::user();
-        $lang = config('lcm.lang.app_local');
-        $supportLang = config('lcm.lang.support_local');
+        $lang = config('lcm.app_locale');
+        $supportLang = config('lcm.supported_locales');
+
         if (! Auth::check()) {
+
             if (! is_null($request->server('HTTP_ACCEPT_LANGUAGE'))) {
                 $navigatorLang = substr($request->server('HTTP_ACCEPT_LANGUAGE'), 0, 2);
```

Comment:
tu as une erreur phpstan concernant la fonction `substr` qui veut recevoir une chaine de caractère. Faut forcer son type

```php
substr((string) $request->server('HTTP_ACCEPT_LANGUAGE'), 0, 2)
```

Reply:
je l'avais fais mais il affichait l'erreur suivant 
 Cannot cast array|string to string.

Reply:
Rajoute alors un `// @phpstan-ignore-line` à coté de la ligne là